easybuild/easyconfigs/c/CGAL/CGAL-4.6-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,29 @@
+name = 'CGAL'
+version = '4.6'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+sources = [SOURCE_TAR_BZ2]
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/34150/']
+
+pythonversion = '2.7.9'
+versionsuffix = "-Python-%s" % pythonversion
+
+dependencies = [
+#GMP does not compile correctly with ICC on Haswell architecture
+    ('GMP', '6.0.0a', '', ('GCC', '4.9.2')),
+    ('Boost', '1.58.0', versionsuffix),
+    ('MPFR', '3.1.2', '-GMP-6.0.0a'),
+    ('Qt', '4.8.6', '-GLib-2.44.0'),
+]
+
+builddeps = [
```

Comment:
to check what? typos? several easyconfig files use 'local variables'

if the typo is close enough (e.g. `buildependencies`), you'll get an error already:

```
ERROR: Failed to process easyconfig /tmp/bzip2-1.0.6.eb: You may have some typos in your easyconfig file: sources_urls -> source_urls
```